resources/test/conftest.py

```diff
@@ -32,33 +33,58 @@ def pytest_configure(config):
     config.add_cleanup(config.server.stop)
     config.add_cleanup(config.driver.quit)
 
+def resolve_uri(context, uri):
```

Comment:
Nit: "url"

Reply:
This value is a relative or root-relative path, not a URL.